tests/conftest.py

```diff
@@ -33,10 +37,6 @@
 environ['KFPMETA_OUT_DIR'] = f'{results}/kfp/'
```

Comment:
@tebeka this have to come AFTER the import (import uses the env)

Reply:
@tebeka `import mlrun` reads the `KFPMETA_OUT_DIR` env value, so u cant set it up after mlrun was imported

Reply:
I want `mlrun` to use the one set by the environment variables set by `conftest.py`. Since `conftest.py` is the first file `pytest` imports - this affects the environment before we import.